scenario/containertab.go

```diff
@@ -115,20 +128,46 @@ func (settings ContainerTabSettings) Execute(sessionState *session.State, action
 		activeID = settings.ObjectID
 	case ContainerTabModeRandom:
 		idx := sessionState.Randomizer().Rand(len(containerInstance.Children))
-		activeID = containerInstance.Children[idx].ObjID
+		child := containerInstance.Children[idx]
+		if child.External {
+			sessionState.LogEntry.Log(logger.WarningLevel, "")
```

Comment:
Empty log line? Also, is this not done as well in the isExternal method?

Reply:
... goland not working very good in wsl and didn't save it altough I pressed save... will update

Reply:
fixed